e2etest/layer2_test.go

```diff
@@ -73,7 +73,7 @@ var _ = ginkgo.Describe("L2", func() {
 
 		hostport := net.JoinHostPort(ingressIP, port)
 		address := fmt.Sprintf("http://%s/", hostport)
-		err = wgetRetry(false, address)
+		err = wgetRetry("", address)
```

Comment:
Can we split the signature change on a different commit? For sure the next time will help me to review, but is simpler also to catch bugs if commits are atomic and do only one "logical" thing.

Reply:
Will do!

Reply:
Done